acoustid/future/fpindex/client.py

```
@@ -0,0 +1,278 @@
+        """Check if fingerprint exists
+
+        Args:
+            index_name: Name of the index
+            fingerprint_id: ID of the fingerprint
+
+        Returns:
+            True if fingerprint exists, False otherwise
+        """
+        try:
+            status, _ = await self._request(
+                "HEAD", f"/{index_name}/{fingerprint_id}", expected_status=[200, 404]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)